src/components/ClickableStyle/ClickableStyle.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import type { StoryObj, Meta } from '@storybook/react';
+import React from 'react';
+
+import { ClickableStyle, STATUSES, VARIANTS } from './ClickableStyle';
+
+export default {
+  title: 'Components/ClickableStyle',
+  component: ClickableStyle,
+  args: {
+    status: 'brand',
+    variant: 'primary',
+    as: 'a',
+  },
+  parameters: {
+    layout: 'centered',
+    controls: { expand: true },
+  },
+  argTypes: {
+    status: {
+      options: STATUSES,
+      control: { type: 'radio' },
+    },
+    variant: {
+      options: VARIANTS,
+      control: { type: 'radio' },
+    },
+    as: {
+      options: ['a', 'button'],
+      control: { type: 'radio' },
+    },
+  },
+} as Meta<Args>;
+
+type Args = React.ComponentProps<typeof ClickableStyle>;
+
+export const Default: StoryObj<Args> = {
+  args: {
+    children: 'link',
+  },
+};
+
+type ExampleProps = {
+  to: string;
+};
+const CustomElement = ({ to, ...rest }: ExampleProps) => (
```

Comment:
This is clear in context, but to document it... the custom element here is arbitrary, and not a specific example of what can be don with `as`. But deliberately mimics what a router link from some library might look like